include/libpmemobj++/detail/common.hpp

```diff
@@ -50,6 +50,19 @@
 #define POBJ_CPP_DEPRECATED
 #endif
 
+/*
+ * Workaround for missing "is_trivially_copyable" in gcc < 5.0.
+ * Be aware of a difference between __has_trivial_copy and is_trivially_copyable
+ * e.g. for deleted copy constructors __has_trivial_copy(A) returns 1 in clang
+ * and 0 in gcc. It means that for gcc < 5 IS_TRIVIALLY_COPYABLE is more
+ * restrictive than is_trivially_copyable.
+ */
+#if defined(__GNUG__) && __GNUC__ < 5
```

Comment:
Are you sure this won't trigger on clang? IIRC clang impersonates gcc on Linux (and msvc on Windows)...